lib/octokit/client/stats.rb

```diff
@@ -75,11 +75,26 @@ def punch_card_stats(repo, options = {})
       #
       # @param repo [Integer, String, Hash, Repository] A GitHub repository
       # @param metric [String] The metrics you are looking for
-      # @return [Array<Sawyer::Resource>] Magical unicorn stats
+      # @return [Array<Sawyer::Resource> or nil] Magical unicorn stats or nil if unicorn haven't calculated stats yet
```

Comment:
Ugh, the original return type comment here is bad. We should make this:

> Stats in metric-specific format, or nil if not yet calculated.

... and add a `@see https://developer.github.com/v3/repos/statistics/`